docs/reference/elasticsearch/rest-apis/reindex-indices.md

```diff
@@ -0,0 +1,630 @@
+---
+navigation_title: "Reindex indices"
+mapped_pages:
+  - https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-reindex.html
+applies_to:
+  stack: all
+---
+
+# Reindex indices examples
+
+::::{admonition} New API reference
+For the most up-to-date API details, refer to [Document APIs](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-reindex).
+::::
```

Comment:
```suggestion
This page provides examples of how to use the [Reindex API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-reindex).
```

Reply:
It would be good to provide a jumplist like @kosabogi did in her PR too so readers can scan what they can learn from the page :)

Reply:
@charlotte-hoblik, I think you're right, the jumplist here is too dominating, but perhaps a short overview of the kinds of examples users can find here, in prose, would be be a bit more user friendly :)